mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +27,75 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enlarge Image
+   */
+  isZoomable?: boolean;
+
+  /**
+   * Enlarge Image Styles
+   */
+  enlargeClassName?: string;
+
+  /**
+   * Enlarge Image Styles
+   */
+  isDisableZoom?: boolean;
 }
 
 /**
  * Сomponent for displaying images
  */
 export const Image = (props: ImageProps) => {
-  const className = clsx(styles.image, props.className);
+  const {src, alt, dataCy, className, enlargeClassName, isZoomable, isDisableZoom} = props;
 
-  return (
+  const [zoomable, setZoomable] = useState(isZoomable);
+
+  useEffect(() => {
+    if (isDisableZoom) {
+      setZoomable(false);
+    } else {
+      setZoomable(isZoomable);
+    }
+  }, [isDisableZoom, isZoomable]);
+
+  const combinedClassName = clsx(styles.image, className);
+  const combinedEnlargeClassName = clsx(styles.image, enlargeClassName);
+
+  const imageElement = (
+    <img
+      src={src}
+      alt={alt}
+      className={combinedClassName}
+      data-cy={dataCy}
+    />
+  );
+
+  const enlargedImageElement = (
     <img
-      src={props.src}
-      alt={props.alt}
-      className={className}
-      data-cy={props.dataCy}
+      src={src}
+      alt={alt}
+      className={combinedEnlargeClassName}
+      data-cy={dataCy}
```

Comment:
if we want to make one of props optional we don't need to programm two whole components

Reply:
I still can't resolve out how to solve this problem